### DISCUSSION TITLE
EMQX Publisher/Subscriber and message through put limit

### DISCUSSION THREAD

**Discussion by deveshmishra34** [emqx/emqx#11179] [General]
Can someone share detail. Limit of Publisher/Subscriber and message through put per topic

**Comment by lafirest** [comment]
@deveshmishra34 
Currently, EMQX doesn't support limiting the rate on topic, but the workaround is possible, you can use a ACL backend to do this. For example, a webhook backend use fixed window algorithm to refused the publish/subscribe which exceed the fixed window.
This is just working, but not perfect.
